tools/src/main/java/org/apache/kafka/tools/ConnectPluginPath.java

```diff
@@ -368,6 +391,30 @@ private static void endCommand(
             config.out.printf("Total plugins:      \t%d%n", totalPlugins);
             config.out.printf("Loadable plugins:   \t%d%n", loadablePlugins);
             config.out.printf("Compatible plugins: \t%d%n", compatiblePlugins);
+        } else if (config.command == Command.SYNC_MANIFESTS) {
+            if (workspace.commit(true)) {
+                if (config.dryRun) {
+                    config.out.println("Dry run passed: All above changes can be committed to disk if re-run with dry run disabled.");
+                } else {
+                    config.out.println("Writing changes to plugins...");
+                    workspace.commit(false);
+                    config.out.println("All plugins have accurate ServiceLoader manifests");
```

Comment:
Nit (I know this may be paranoid but I still think people are going to be confused/frustrated if they see this message but the script couldn't migrate a broken plugin):
```suggestion
                    config.out.println("All loadable plugins have accurate ServiceLoader manifests");
```